Temp fs dsim mapping update

AWh1t3 opened the pull request:
Updating Jan's land-sim to ndlar-flow channel ![image](https://github.com/user-attachments/assets/8aca95d9-50e5-4dde-b1f0-9cda6cffa580)

AWh1t3 committed:
Delete yamls/fsd_flow/reco/light/LightEventGeneratorMC.yaml

delete old

AWh1t3 committed:
Delete yamls/fsd_flow/reco/light/LightADC64EventGenerator.yaml

delete old

AWh1t3 committed:
Add files via upload

AWh1t3 committed:
Update LightEventGeneratorMC.yaml

Updated channel mapping: matches https://docs.google.com/spreadsheets/d/1YnXpoeFB6laCJ_2x0_RqJg6IHxhZB9kz6KCtvWv5f9Q/edit?gid=1754157014#gid=1754157014, assumes larndsim channel mapping follows https://docs.google.com/spreadsheets/d/1G4LgfH1r7tTLs5DAgaJuRqW1nV-9XD-EU-Tr_UWgW4g/edit?gid=0#gid=0

AWh1t3 committed:
Update LightEventGeneratorMC.yaml

fixed formatting

AWh1t3 committed:
Update LightEventGeneratorMC.yaml

AWh1t3 committed:
Update LightEventGeneratorMC.yaml

AWh1t3 committed:
Update LightEventGeneratorMC.yaml

krwood commented:
After discussing with @AWh1t3 on SLACK, we will hold of on merging this PR until some tests are run later this week / early next week. We will use those tests to validate the above changes. Please upload results to this thread when they are available. (cc @cuddandr)

AWh1t3 committed:
Update LightEventGeneratorMC.yaml

diaza committed:
Load geometry in the first stage for FSD

cuddandr committed:
Merge pull request #175 from DUNE/fsd_disabledchannelhotfix

Load geometry in the first stage for FSD

cuddandr commented:
With Alex's fix for loading/skipping the dead channels map everything is working as far as I can tell on my end. Did any of the validation mentioned above happen? I can also provide the cosmic sim flow file I managed to produce using the this branch.

AWh1t3 commented:
If you send me the location of the file, I can do a quick validation! Jan might have validated the branch already, but I haven't heard anything.

cuddandr commented:
FSD cosmic sim flow file with the new light noise spectra can be found here (also sent in a Slack message): ```/global/cfs/projectdirs/dune/www/data/FSD/simulation/FSD_CosmicRun1_lightnoise.flow.0000000.FLOW.hdf5```

cuddandr committed:
Further update FSD yamls for light simulation (remove deconv step)

Remove the waveform deconvolution step for now as the FSD specific
inputs are not available.

Fix a typo in the channel map in `LightEventGeneratorMC.yaml`

Change a few remaining 2x2 values to FSD values (e.g. nchantpc)

cuddandr commented:
I'm trying to get everything organized for the FSD production with light and would like to finish merging this in to develop. I produced several flow files with the light simulation both from this branch and with a test merge into develop and nothing seems obviously wrong looking at a few entries in the hdf5 files.

Are there any further checks we want to do now, or merge this and do it as part of an initial validation of the FSD production (where I produce ~30 or so files).

Current files available here: `/global/cfs/projectdirs/dune/www/data/FSD/simulation/`. The indices 0 and 1 files are just this branch, and 2 and 3 are including the latest from `develop` on top of this branch.

diaza commented:
Looks good to me. But I see there are still merge conflicts?

cuddandr commented:
> Looks good to me. But I see there are still merge conflicts?

Yeah, those are from Noe and me editing the same yamls. I'll fix them as part of the merge / closing the PR. I already tested the changes in a local pull as part of producing a couple files.

krwood commented:
Looks good to me too. It looks like some of these yaml files are used for processing both MC and data. One more quick check would be to verify that these changes run of data files as expected.

krwood commented:
@cuddandr are you still planning on resolving the conflicts and merging in?
